roles/bootstrap-os/defaults/main.yml

```diff
@@ -3,7 +3,6 @@ bootstrap_os:
 os_family: "{{bootstrap_os}}"
 
 pip_python_coreos_modules:
-  - httplib2
```

Comment:
This should not be changed.
Instead maybe define a `pip_python_oracle_modules` variable and use it the same way

Reply:
Will do. Thanks